packages/angular/src/migrations/update-14-8-0/rename-webpack-server.ts

```diff
@@ -0,0 +1,22 @@
+import type { Tree } from '@nrwl/devkit';
+import { getProjects, joinPathFragments, writeJson } from '@nrwl/devkit';
+
+export default function renameWebpackServer(tree: Tree) {
+  const projects = getProjects(tree);
+
+  for (const [projectName, project] of projects.entries()) {
+    let stringifiedProject = JSON.stringify(project);
+    if (stringifiedProject.includes('@nrwl/angular:webpack-server')) {
+      stringifiedProject = stringifiedProject.replace(
+        /@nrwl\/angular:webpack-server/g,
+        '@nrwl/angular:webpack-dev-server'
+      );
+
+      writeJson(
+        tree,
+        joinPathFragments(project.root, 'project.json'),
+        JSON.parse(stringifiedProject)
+      );
```

Comment:
We should avoid explicitly modifying `project.json` files (or any other configuration file by name). We should instead use the likes of `updateProjectConfiguration` and `updateWorkspaceConfiguration` . The structure and names of the configuration files are always bound to change (though unlikely to happen too often), so it's better to stick with the DevKit utilities when working with the configuration.

Also, we probably want to avoid serializing and deserializing the config as well, better to refactor this to go over all the target executors and change them as needed (should be simple as well).